.travis.yml

```diff
@@ -2,21 +2,22 @@ language: rust
 rust:
   - stable
   - nightly
-
+os:
+  - linux
+  - osx
+  - windows
 env:
-  - NODE_ENV=ci
-  - NODE_ENV=ci-staging
-cache: yarn
-
+  - NODE_ENV=ci YARN_GPG=no
+  - NODE_ENV=ci-staging YARN_GPG=no
+cache: cargo
```

Comment:
Is there a reason to stop caching `yarn`?

Reply:
good question! I can't confirm this 100% but I haven't seen multiple `cache` keys in a Travis yaml online yet. The only time I've seen the `cache: yarn` is when the "primary" (if that's the term?) `language` declared at the top if `node_js` like shown here.
https://docs.travis-ci.com/user/caching/#yarn-cache
When I was watching Travis like a hawk, I noticed the cargo stuff took forever. I thought I might as well cache cargo here:
https://docs.travis-ci.com/user/languages/rust/#dependency-management
I believe this choice was somewhat informed, and somewhat, "this is how I've seen it elsewhere online" as well as a smidge of guesswork. Perhaps we can tinker with trying multiple cache keys in a WIP branch? (Really wish I could test Travis stuff locally but looking online that doesn't seem reliable/well-supported)

Reply:
fair enough